pkg/cmd/cli/cmd/deploy.go

```diff
@@ -284,14 +286,17 @@ func (o *DeployOptions) cancel(config *deployapi.DeploymentConfig, out io.Writer
 		return err
 	}
 	if len(deployments.Items) == 0 {
-		fmt.Fprintln(out, "no deployments found to cancel")
+		fmt.Fprintf(out, "there have been no deployments for %s/%s\n", config.Namespace, config.Name)
```

Comment:
We use lowercase words in human readable messages.

On Sep 15, 2015, at 8:58 AM, Dan Mace <notifications@github.com> wrote:

In pkg/cmd/cli/cmd/deploy.go
<https://github.com/openshift/origin/pull/4669#discussion_r39505925>:

> @@ -284,14 +286,17 @@ func (o *DeployOptions) cancel(config *deployapi.DeploymentConfig, out io.Writer
>  		return err
>  	}
>  	if len(deployments.Items) == 0 {
> -		fmt.Fprintln(out, "no deployments found to cancel")
> +		fmt.Fprintf(out, "there have been no deployments for %s/%s\n", config.Namespace, config.Name)

Not sure. They have always been lower case here, for whatever reason.
@smarterclayton <https://github.com/smarterclayton>?

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/4669/files#r39505925>.